core/src/com/unciv/logic/automation/UnitAutomation.kt

```diff
@@ -268,6 +270,25 @@ object UnitAutomation {
         return true
     }
 
+    private fun tryHeadTowardsSiegedCity(unit: MapUnit): Boolean {
+        val siegedCities = unit.civInfo.cities
+                .asSequence()
+                .filter { unit.civInfo == it.civInfo &&
+                        it.health < (it.getMaxHealth() * 0.75) && //Weird health issues and making sure that not all forces move to good defenses
+                        it.isInResistance() } //Don't stop pushing keep the battle tempo up
+
+        val strongestForce = siegedCities
+                .flatMap { it.getCenterTile().getTilesAtDistance(2) }
+                .sortedByDescending { it.getInfluence() }
```

Comment:
No, this should definitely be sorted by descending aerial distance.
Time is of essence with this city, since resistance doesn't last long, we want it to get into action as soon as possible